<ui version="4.0"> 
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Civilization</string>
  </property>
    <property name="windowIcon">
   <iconset>./res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" />
    <property name="autoFillBackground">
      <bool>true</bool>
    </property>
  <widget class="QStackedWidget" name="stackedWidget">
    <property name="autoFillBackground">
      <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>640</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
      <widget class="QLabel" name="start_movie">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>0</y>
            <width>960</width>
            <height>640</height>
          </rect>
        </property>
        <property name="autoFillBackground">
          <bool>true</bool>
        </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
          <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>500</height>
          </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
          <number>6</number>
        </property>
        <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
          <widget class="QPushButton" name="B_start">
          <property name="autoFillBackground">
            <bool>true</bool>
          </property>
          <property name="styleSheet">
            <string notr="true"> background-color: rgb(255, 255, 255,0);</string>
            </property>
          <property name="icon">
          <iconset resource="image.qrc">
            <normaloff>./res/start/start_selected.png</normaloff>./res/start/start_selected.png</iconset>
          </property>
          <property name="text">
            <string/>
          </property>
          <property name="iconSize">
            <size>
              <width>300</width>
              <height>200</height>
            </size>
          </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="B_settings">
          <property name="autoFillBackground">
            <bool>true</bool>
          </property>
          <property name="text">
            <string>SETTINGS</string>
          </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="B_quit">
          <property name="autoFillBackground">
            <bool>true</bool>
          </property>
          <property name="text">
            <string>QUIT</string>
          </property>
          </widget>
        </item>
        </layout>
      </widget>
   
     </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
